### ISSUE TITLE
Missing simple test

### ISSUE THREAD

**Issue by martymcguire** [adafruit/Adafruit_CircuitPython_asyncio#45]
The [Examples for asyncio on Read the Docs](https://docs.circuitpython.org/projects/asyncio/en/latest/examples.html) have a `examples/asyncio_simpletest.py` with some partial comments and no other content.

Indeed, the entire contents in that file in this repo since its creation are:

```
# SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
# SPDX-FileCopyrightText: Copyright (c) 2021 Dan Halbert for Adafruit Industries
#
# SPDX-License-Identifier: Unlicense
```

Would love an example to follow that doesn't require a whole PyPortal! :}

**Comment by dhalbert** [comment]
The guide has a lot of examples: https://learn.adafruit.com/cooperative-multitasking-in-circuitpython-with-asyncio. We should probably point to that.

**Comment by FoamyGuy** [comment]
resolved by #67
